### ISSUE TITLE
Testing for Marker Colocalization

### ISSUE THREAD

**Issue by ikuznet1** [Upward-Spiral-Science/the-fat-boys#4]
I have been trying to see which markers colocalize best with GABABR. I have followed the standard method of looking at colocalization by taking the Pearson correlation between the raw fluorescence images. This method provides interesting results but interpretability is difficult due to the large quantity of noise.

What I have tried to do instead is the following:
Use non-max suppression on the raw data to extract all local maximums in the image above a certain threshold.  This leaves me with a 2D scatterplot of "synapse" point locations for each channel. I now want to compare the scatterplot for GABABR to the other marker's 2D scatterplots for "similiarity". I have thought of 2 ways to do this so far:
1) Pearson Correlation between them. This works poorly because it compares them pixel by pixel. It ignores points that are close-by but not quite overlapping.
2) Find the distance from each point in the GABABR scatterplot and its nearest neighbor in the scatterplot being compared to. Find the median and compare across different markers. The issue here is that the spread of the medians across the different markers is minimal so it is hard to draw conclusions.

Are there better ways of doing this comparison? Perhaps some sort of variation on the Wilcoxon Rank Sum Test to make it work for bivariate data? This paper seems to propose this: http://tbiomed.biomedcentral.com/articles/10.1186/1742-4682-7-13

**Comment by jovo** [comment]
You got satisfaction in class?

On Thursday, April 21, 2016, ikuznet1 notifications@github.com wrote:

> I have been trying to see which markers colocalize best with GABABR. I
> have followed the standard method of looking at colocalization by taking
> the Pearson correlation between the raw fluorescence images. This method
> provides interesting results but interpretability is difficult due to the
> large quantity of noise.
> 
> What I have tried to do instead is the following:
> Use non-max suppression on the raw data to extract all local maximums in
> the image above a certain threshold. This leaves me with a 2D scatterplot
> of "synapse" point locations for each channel. I now want to compare the
> scatterplot for GABABR to the other marker's 2D scatterplots for
> "similiarity". I have thought of 2 ways to do this so far:
> 1) Pearson Correlation between them. This works poorly because it compares
> them pixel by pixel. It ignores points that are close-by but not quite
> overlapping.
> 2) Find the distance from each point in the GABABR scatterplot and its
> nearest neighbor in the scatterplot being compared to. Find the median and
> compare across different markers. The issue here is that the spread of the
> medians across the different markers is minimal so it is hard to draw
> conclusions.
> 
> Are there better ways of doing this comparison? Perhaps some sort of
> variation on the Wilcoxon Rank Sum Test to make it work for bivariate data?
> This paper seems to propose this:
> http://tbiomed.biomedcentral.com/articles/10.1186/1742-4682-7-13
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> https://github.com/Upward-Spiral-Science/the-fat-boys/issues/4

## 

the glass is all full:  half water, half air.
neurodata.io

**Comment by ikuznet1** [comment]
Yes. I'm implementing the methods described in the paper. I'll see where
that takes me.

**Comment by jovo** [comment]
ok

On Thu, Apr 21, 2016 at 7:31 PM, ikuznet1 notifications@github.com wrote:

> Yes. I'm implementing the methods described in the paper. I'll see where
> that takes me.
> On Apr 21, 2016 7:26 PM, "joshua vogelstein" notifications@github.com
> wrote:
> 
> > You got satisfaction in class?
> > 
> > On Thursday, April 21, 2016, ikuznet1 notifications@github.com wrote:
> > 
> > > I have been trying to see which markers colocalize best with GABABR. I
> > > have followed the standard method of looking at colocalization by
> > > taking
> > > the Pearson correlation between the raw fluorescence images. This
> > > method
> > > provides interesting results but interpretability is difficult due to
> > > the
> > > large quantity of noise.
> > > 
> > > What I have tried to do instead is the following:
> > > Use non-max suppression on the raw data to extract all local maximums
> > > in
> > > the image above a certain threshold. This leaves me with a 2D
> > > scatterplot
> > > of "synapse" point locations for each channel. I now want to compare
> > > the
> > > scatterplot for GABABR to the other marker's 2D scatterplots for
> > > "similiarity". I have thought of 2 ways to do this so far:
> > > 1) Pearson Correlation between them. This works poorly because it
> > > compares
> > > them pixel by pixel. It ignores points that are close-by but not quite
> > > overlapping.
> > > 2) Find the distance from each point in the GABABR scatterplot and its
> > > nearest neighbor in the scatterplot being compared to. Find the median
> > > and
> > > compare across different markers. The issue here is that the spread of
> > > the
> > > medians across the different markers is minimal so it is hard to draw
> > > conclusions.
> > > 
> > > Are there better ways of doing this comparison? Perhaps some sort of
> > > variation on the Wilcoxon Rank Sum Test to make it work for bivariate
> > > data?
> > > This paper seems to propose this:
> > > http://tbiomed.biomedcentral.com/articles/10.1186/1742-4682-7-13
> > > 
> > > —
> > > You are receiving this because you are subscribed to this thread.
> > > Reply to this email directly or view it on GitHub
> > > https://github.com/Upward-Spiral-Science/the-fat-boys/issues/4
> > 
> > ## 
> > 
> > the glass is all full: half water, half air.
> > neurodata.io
> > 
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly or view it on GitHub
> > <
> > https://github.com/Upward-Spiral-Science/the-fat-boys/issues/4#issuecomment-213157644
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> https://github.com/Upward-Spiral-Science/the-fat-boys/issues/4#issuecomment-213159158

## 

the glass is all full:  half water, half air.
neurodata.io
